tests/phpunit/diagnostics/class-wc-stripe-diagnostics-cleanup-test.php

```
@@ -0,0 +1,103 @@
+	 *
+	 * @param string $id         Session id of the trace to mutate.
+	 * @param int    $created_at New `created_at` epoch seconds.
+	 * @param int    $updated_at New `updated_at` epoch seconds.
+	 * @return void
+	 */
+	private function set_times( string $id, int $created_at, int $updated_at ): void {
+		$trace               = $this->store->get( $id );
+		$trace['created_at'] = $created_at;
+		$trace['updated_at'] = $updated_at;
+		file_put_contents( $this->store->get_trace_path( $id ), wp_json_encode( $trace ) ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_file_put_contents
+	}
```

PHPMD (2.15.0)

[error] The method set_times is not named in camelCase. (undefined)

(CamelCaseMethodName)